mdsuite/experiment/experiment.py

```diff
@@ -40,6 +41,49 @@
 log = logging.getLogger(__file__)
 
 
+class RunModule:
```

Comment:
This is nice but can we move it to a separate module and import it? leave it within the experiment directory but make a new file. We can of course discuss this but my thoughts are that we separate modules by their directory structure and so we can spread out class declaration within these directories. As the experiment class is very big and has a lot to do, it might be best to move the computation handling to its own module.

Reply:
Yeah, sure. I also thought about it, where do you think it would fit? Probably into utils?

Reply:
I would still leave it in the experiment directory. Just move it to its own module.

Reply:
fixed in d17c8a1c53dd9ed64ebcaafd63a498cc3198f412